templates/autoscale-existing-vpc.template

```diff
@@ -0,0 +1,1010 @@
+        },
+        "LoadBalancingHealthCheckThreshold": {
+            "Type": "Number",
+            "Default": 3,
+            "MinValue": 3,
+            "ConstraintDescription": "must be a valid number not less than 3.",
+            "Description": "The number of consecutive health check failures required before considering a FortiGate instance unhealthy. Minimum is 3."
+        },
+        "InternalLoadBalancingOptions": {
+            "Type": "String",
+            "Default": "add a new internal load balancer",
+            "AllowedValues": [
+                "add a new internal load balancer",
+                "use the load balancer specified below",
+                "do not need one"
+            ],
+            "ConstraintDescription": "must choose from the provided options.",
+            "Description": "(Optional) The pre-defined load balancer to route traffic to web service in the private subnets."
+        },
+        "InternalTargetGroupHealthCheckPath": {
+            "Type": "String",
+            "Default": "/",
+            "AllowedPattern": "^/[0-9a-zA-Z-/]*$",
+            "MaxLength": 1024,
+            "ConstraintDescription": "This path must begin with a '/' character, and can be at most 1024 characters in length.",
+            "Description": "(Optional) The destination path for health checks. This path must begin with a '/' character, and can be at most 1024 characters in length."
+        },
+        "InternalLoadBalancerDNSName": {
+            "Type": "String",
+            "Default": "",
+            "Description": "(Optional) The DNS name of an existing internal load balancer used to route traffic from a FortiGate to targets in a specified target group. Leave it blank if you don't use an existing load balancer."
+        },
+        "GetLicenseGracePeriod": {
+            "Type": "Number",
+            "Default": 600,
+            "MinValue": 300,
+            "ConstraintDescription": "must be a valid number not less than 300.",
+            "Description": "The minimum time (in seconds) permitted before a distributed license can be revoked from a non-responsive FortiGate and re-distributed. Minimum is 300."
+        },
+        "FortiAnalyzerIntegrationOptions": {
+            "Type": "String",
+            "Default": "yes",
+            "AllowedValues": ["yes", "no"],
+            "ConstraintDescription": "must choose from the provided options.",
+            "Description": "Choose 'yes' to incorporate FortiAnalyzer into Fortinet FortiGate Auto Scaling to use extended features that include storing logs into FortiAnalyzer."
+        },
+        "FortiAnalyzerInstanceType": {
+            "Type": "String",
+            "Default": "m4.large",
+            "AllowedValues": [
+                "c4.large",
+                "c4.2xlarge",
+                "d2.4xlarge",
```

Comment:
Should this be c4?

Reply:
will find out after I added a full list of supported instance types in a follow change.

Reply:
@JaydenLiang

Reply:
fixed in dd81623ad6558d6a1cdd7b4c98a7fdf0ce054e86

Reply:
do not need this change. please delete this suggestion. thanks